pgconfeu-recap.mdx

```diff
@@ -0,0 +1,180 @@
+
+If you are interested in joining the Diversity Task Force, send us an email at diversity@postgresql.eu.
+
+## PostgreSQL Extension Ecosystem Summit & pgzx Talk
+
+After the PostgreSQL Europe Diversity Committee meeting, I've attended PostgreSQL Extension Ecosystem Summit, and gave talk about [pgzx](https://github.com/xataio/pgzx).
+
+Zig is a small and simple language that aims to be a "modern C" and make system-level code bases easier to maintain. It provides safe memory management, compilation time code execution (comptime), and a rich standard library.
+Zig can interact with C code quite naturally: it supports the C ABI, can work with C pointers and types directly, it can import header files and even translate C code to Zig code. Thanks to this interoperability, a Postgres extension written in Zig can, theoretically, accomplish anything that a C extension can.
+This means you get full power and a modern language and standard library to write your extension.
+
+While in theory you can write any extension in Zig that you could in C, in practice you will need to make sense of a lot of Postgres internals in order to know how to correctly use them from Zig.
+Also, Postgres makes extensive use of macros, and not all of them can be translated automatically.
+
+This is where pgzx comes in. pgzx is an open-source project that provides a framework for creating PostgreSQL extensions using the Zig programming language.
+The goal of pgzx is to simplify the process of writing PostgreSQL extensions by leveraging the simplicity and performance benefits of Zig.
+
+What pain points does pgzx address beyond simply enabling Zig development?
+
+- Easy Integration with Existing C Code: pgzx allows developers to work seamlessly with existing C code without imposing too many abstractions.
+- Handling Postgres C Macros: Postgres code relies heavily on C macros, which do not always compile in Zig. Rather than trying to create an abstraction layer over Postgres, the aim is to use the C Postgres API as much as possible. If limitations arise—such as macros not compiling or issues with type checking—we provide solutions through pgzx.
+- Collaborative Approach: pgzx fosters a collaborative integration with Postgres, working alongside it rather than imposing layers on top or acting against it.
+- A New Way to Build Postgres Extensions
+
+If you are interested in learning more, you can visit the [pgzx GitHub repository](https://github.com/xataio/pgzx) and take a look at the sample [extensions](https://github.com/xataio/pgzx/tree/main/examples) we've built with it.
+
+You can find my slides [here](https://github.com/tembo-io/tembo-labs/blob/main/extensions-ecosystem-summit/pgzx_gulcin.pdf) and all other talks from the summit [here](https://github.com/tembo-io/tembo-labs/tree/main/extensions-ecosystem-summit).
+
+## Xata Postgres Dinner
```

Comment:
not necessary, but there were some fun photos here. you could drop them in

Reply:
I have too many, need to choose a few :D